### PR TITLE
sanantonio.intro.1.gonzalez.julio

### PR DESCRIPTION
**Pull request by 2104382953**

### CODE REVIEW

File: 02LessonTwo/homework/index.html [hunk 1]
```diff
@@ -5,6 +5,34 @@
   <link rel="stylesheet" href="css/style.css">
 </head>
 <body>
+  <h2>Julio Gonzalez III</h2>
+  <center><img style= width:400px; height:300px; src="https://scontent-dfw1-1.xx.fbcdn.net/v/t1.0-9/10410162_10153082863091789_6841706743737948299_n.jpg?oh=f9b82da217710d3c473f42eef0df02b4&oe=582F4B1B"/></center>
```
Comment (jlug331221) [inline 02LessonTwo/homework/index.html]: Make sure to wrap your inline CSS style properties and values in quotes.
``

File: 02LessonTwo/homework/index.html [hunk 2]
```diff
@@ -5,6 +5,34 @@
...
+      <li>the SEC Conference</li>
+    </ul>
+  </div>
+  <h5>TV Shows I like...</h5>
+  <img style= width:750px; heigth:350px; src="http://www.nbc.com/sites/nbcunbc/files/files/images/2015/2/26/150226_2849965_The_Farewell_Season__Shooting_the_Final_Scen.jpg"/>
+  <img style= width:750px; heigth:350px; src="http://cdn3.pcadvisor.co.uk/cmsdata/features/3627099/walking_dead_season_6_thumb800.jpg"/>
+
 
 </body>
 </html>
```
Comment (jlug331221) [inline 02LessonTwo/homework/index.html:45]: Good job on your bio page Julio. Keep up the good work.

File: 03LessonThree/homework/git-scm.html [hunk 3]
```diff
@@ -30,31 +33,31 @@
...
                 <h3>Community</h3>
                 <p>Questions or comments for the Git community can be sent to the mailing list by using the email address git@vger.kernel.org. Bug reports should be sent to this mailing list.</p>
                 <p>You do not need to subscribe: you will be Cc'd in replies. Please keep the Cc list intact when replying (use "Reply to all"). Greylisting may delay your first post for a few hours. Note also that the list only accepts plain-text email; please disable HTML in your outgoing messages.</p>
                 <p>The archive can be found on Gmane. Click here to subscribe. If you have trouble sending to the list with your regular mail reader, you may also post via gmane. There is also Git user mailing list on Google Groups which is a nice place for beginners to ask about anything.</p>
-            </div>
-        </div>
-        <div id="aside">
-            <div>
+            </article>
+        </section>
```
Comment (jlug331221) [inline 03LessonThree/homework/git-scm.html:43]: For lines 26-43, remove the extra tab spacing. Your opening `<section>` tag should be lined up with the closing `</header>` tag. I know we really have not talked about code formatting but I can go over this today with you and the rest of the class.

File: 03LessonThree/homework/git-scm.html [hunk 4]
```diff
@@ -30,31 +33,31 @@
...
-    <div id="footer">
+
+        </aside>
+    <footer>
         This open sourced site is hosted on GitHub.<br/> Patches, suggestions and comments are welcome.
-    </div>
+    </footer>
 </body>
 
 </html>
```
Comment (jlug331221) [inline 03LessonThree/homework/git-scm.html:63]: Good work on this assignment! I like how you added the extra CSS to center the content and add borders.

File: 05LessonFive/Checkpoint1/css/style.css [hunk 5]
```diff
@@ -21,14 +30,15 @@
   padding-left: 5px;
   padding-right: 5px;
   padding-bottom: 5px;
```
Comment (jlug331221) [inline 05LessonFive/Checkpoint1/css/style.css:32]: Padding can be done all on one line if all sides have the same padding. 
Ex: 
`padding: 5px;`

File: 05LessonFive/Checkpoint1/index.html [hunk 6]
```diff
@@ -9,7 +9,7 @@
   <div class="container">
 
     <section id="leftBox">
-      <img src="img/berries.jpg"style="width:100%;"/>
+      <img src="img/berries.jpg"/>
     </section>
 
     <section id="rightBox">
```
Comment (jlug331221) [inline 05LessonFive/Checkpoint1/index.html:15]: Great job Julio, keep up the good work!

File: 07LessonSeven/homework/index.html [hunk 7]
```diff
@@ -5,6 +5,28 @@
...
+          <div>Sign up as company</div>
+        </section>
+        <section class="box2">
+          <h1>Looking for a job</h1>
+          <p>Vivamus congue rhoncus sem et placerat. Fusce nec nunc lobortis lorem ultrices facilisis.Ut dapibus blandit nunc, et consectetur dui.</p>
+        </section>
+      </section>
+  </section>
 </body>
 </html>
```
Comment (jlug331221) [inline 07LessonSeven/homework/index.html:46]: It looks like you were off to a good start for this assignment. You seem to have all the necessary content (except for the "Sign up as an employee" text) but the finished product is not done. Keep working on it and submit another commit when you are done.

Comment (2104382953) [inline 07LessonSeven/homework/index.html:46]: will do

File: 08LessonEight/homework/exercise-06.html [hunk 8]
```diff
@@ -1,10 +1,30 @@
...
+      <span>Read More</span>
+    </section>
 
-<body></body>
-
-</html>
\ No newline at end of file
+  </section>
+</body>
+</html>
```
Comment (jlug331221) [inline 08LessonEight/homework/exercise-06.html:30]: Great job on this assignment Julio. The only thing I would say to work on would be buttons. They also need to line up horizontally. Keep up the good work!

File: 10LessonTen/Checkpoint2/index.html [hunk 9]
```diff
@@ -6,5 +6,31 @@
...
+        <p>Etiam pharetra, erat sed fermentum frugiat, velit mauris egestas quam, ut aliquam massa nisl quis neque. Suspendisse in orci enim.</p>
+      </article>
+      <article>
+        <p> <a href="service.html"><button type="button" style="width:100px; height:50px;"name="CONTACT US ">CONTACT US ></button></a> </p>
+      </article>
+    </section>
+
+  </div>
 </body>
 </html>
```
Comment (jlug331221) [inline 10LessonTen/Checkpoint2/index.html:55]: It looks like you still need to finish up with the checkpoint 2 assignment. I will grade it when you are finished.

File: 11LessonEleven/homework/css/inspired.css [hunk 10]
```diff
@@ -0,0 +1,39 @@
...
+    content: "320px";
+    color: orange;
+  }
+}
+@media screen and (min-width: 480px) {
+  .container::before {
+    content: "480px";
+    color: maroon;
+  }
+}
```
Comment (bbaic) [inline 11LessonEleven/homework/css/inspired.css]: You forgot to change the background image of `.box1`!

File: 11LessonEleven/homework/css/inspired.css [hunk 11]
```diff
@@ -0,0 +1,39 @@
+.container{
+  display: flex;
+  flex-direction: column;
+
+}
+.box1{
```
Comment (bbaic) [inline 11LessonEleven/homework/css/inspired.css:6]: I suggest adding some `padding-left` to all of your `.box#` elements; people don't like reading text that's right at the edge of something.

File: 11LessonEleven/homework/css/inspired.css [hunk 12]
```diff
@@ -0,0 +1,39 @@
...
+  background: #000 url("../img/bg2.jpg") no-repeat right;
+  background-size: cover;
+  height: 400px;
+  color: white;
+  font-size: 1em;
```
Comment (bbaic) [inline 11LessonEleven/homework/css/inspired.css:11]: Why overriding all of the font-sizes? It's okay to do so, just understand why. Also, the sizes you chose are probably too small; don't want to make people squint...

File: 11LessonEleven/homework/css/inspired.css (hunk 10 above)
Comment (bbaic) [inline 11LessonEleven/homework/css/inspired.css]: Also, the two text sections (box2 and box3) are supposed to transition to being side-by-side.

File: 11LessonEleven/homework/css/inspired.css [hunk 13]
```diff
@@ -0,0 +1,39 @@
...
+  .container::before {
+    content: "480px";
+    color: maroon;
+  }
+}
+@media screen and (min-width: 620px) {
+  .container::before {
+    content: "620px";
+    color: orange;
+  }
```
Comment (bbaic) [inline 11LessonEleven/homework/css/inspired.css:39]: You forgot to change the background image of `.box1`!

Comment (bbaic) [inline 11LessonEleven/homework/css/inspired.css:39]: Also, the two text sections (box2 and box3) are supposed to transition to being side-by-side.

File: 12LessonTwelve/homework/css/style.css [hunk 14]
```diff
@@ -0,0 +1,159 @@
...
+    width: 70px;
+  }
+}
+@media screen and (min-width: 810px) {
+  .container::before {
+    content: "810px";
+    color: black;
+  }
+  article {
+    flex-basis: 350px;
```
Comment (bbaic) [inline 12LessonTwelve/homework/css/style.css:143]: Try using a percentage instead. Since you want 4 objects side by side, ~25% (pending margin/padding) should do it.

File: 12LessonTwelve/homework/index.html [hunk 15]
```diff
@@ -5,6 +5,53 @@
...
+              <h2>Happy Ninja</h2>
+              <a href="#">add to cart</a>
+            </div>
+      </article>
+          <article>
+            <div>
+              <div class="img">
+                <img src="https://s3-us-west-2.amazonaws.com/s.cdpn.io/185649/hoodie_1_front.jpg" alt="" />
+                <div class="sale"><span>sale</span></div>
+                <div class="price-change"><span>$15.00</span><b>$12.00</b></div>
```
Comment (bbaic) [inline 12LessonTwelve/homework/index.html:38]: Why not make this class `amount` too? It would go a long way towards a more consistent layout.
Remember, multiple classes use the same `class=` and each class is separated by a space.
ie `class="price-change amount another_class one-more-for-good-measure`

File: 20FinalProject/index.html [hunk 16]
```diff
@@ -3,8 +3,148 @@
...
+      <ol class="carousel-indicators">
+        <li data-target = "#mycarousel" data-slide-to="0" class="active"></li>
+        <li data-target = "#mycarousel" data-slide-to="1"></li>
+        <li data-target = "#mycarousel" data-slide-to="2"></li>
+        <li data-target = "#mycarousel" data-slide-to="3"></li>
+      </ol>
+
+      <div class="carousel-inner">
+          <div class="item active">
+            <a href="https://2104382953.github.io/intro-workbook/12LessonTwelve/in-class-project/"><img src="http://i.imgur.com/Y70OSmC.png" alt="cafe" class="img-responsive" style="width:100%;" /></a>
```
Comment (bbaic) [inline 20FinalProject/index.html:116]: Probably want to add `target="_blank"` to these `a` tags so the user isn't forced to leave your portfolio page.

File: 20FinalProject/index.html [hunk 17]
```diff
@@ -3,8 +3,148 @@
...
+        <span class="icon-prev"></span>
+      </a>
+      <a class="carousel-control right" href="#mycarousel" data-slide ="next">
+        <span class="icon-next"></span>
+      </a>
+    </div>
+
+    <footer>
+      <p class="animated bounceInRight">Find me on Social Media</p>
+      <a href="https://www.facebook.com/julio.gonzalez.ttu"><img src="http://topshelfcateringcompany.com/wp-content/themes/topshelf-child/img/footer-facebook-icon.gif" alt="" /></a>
```
Comment (bbaic) [inline 20FinalProject/index.html:138]: Same note about `target="_blank"`
